src/OrbitGl/include/OrbitGl/BatchRenderGroup.h

```diff
@@ -61,6 +61,8 @@ struct StencilConfig {
   bool enabled = false;
   std::array<float, 2> pos = {0, 0};
   std::array<float, 2> size = {0, 0};
+
+  StencilConfig& ClipAt(const StencilConfig& parent);
```

Comment:
Can we make this a class now and at the trailing `_` to the members?

Reply:
As discussed - I've kept it a struct, but changed the helper method to be a free function.